openedx/core/djangoapps/course_date_signals/handlers.py

```diff
@@ -111,19 +111,22 @@ def extract_dates_from_course(course):
             # unless that item already has a relative date set
             for _, section, weeks_to_complete in spaced_out_sections(course):
                 section_date_items = []
+                max_due_num_weeks = timedelta(weeks=1)
```

Comment:
min? Is this to essentially gate against using anything less than 1? Should this also take into account the max value of 18?

Reply:
The variable is called `max_due_num_weeks` since the variable is reassigned as the maximum number the section could be due in while iterating through the subsections' relative due dates. I assign it as 1 initially since that's the smallest number it could take and when I'm taking the max with other due dates the maximum between the two would actually be taken. I didn't think taking into account the max value of 18 would be necessary since that condition is checked multiple times outside of this function, but if you think this place would be a good extra place to check I can put it!